presto-tests/src/test/java/com/facebook/presto/execution/TestWarnings.java

```diff
@@ -83,6 +83,17 @@ public void testNonReservedWordWarning()
         assertWarnings(queryRunner, session, query, ImmutableList.of(PARSER_WARNING.toWarningCode()));
     }
 
+    @Test
+    public void testNewReservedWordsWarning()
```

Comment:
Also add a test that quoted identifier does not trigger warning.

Reply:
thanks for catching this! Added a test that verifies no warnings for the case of ```"CALLED"``` as an example